server/controllers/auth/login.js

```diff
@@ -3,35 +3,61 @@ const { sign } = require('jsonwebtoken');
 
 const loginValidation = require('../../utils/validation/loginValidation');
 const applicant = require('../../database/models/applicant');
+const admin = require('../../database/models/admin');
+
 require('env2')('config.env');
 
+const erroResponse = {
+  status: 'failed',
+  type: 'wrong Email or Password',
+  erroMessage: 'Check Email or password',
+};
+
 const login = (req, res) => {
   const { email, password } = req.body;
   loginValidation(email, password)
     .then((valid) => {
-      if (!valid) res.status(400).json({ error: 'Check the data you entered' });
+      if (!valid)
+        res.status(400).json({
+          status: 'failed',
+          type: 'type error',
+          erroMessage: 'check the data you inserted',
+        });
       else {
         applicant
           .findOne({ email })
           .then((data) => {
             bcrypt.compare(password, data.password, (err, result) => {
-              if (!result)
-                res.status(400).json({ error: 'wrong Email or password' });
+              if (!result) res.status(400).json(erroResponse);
               else {
                 const userToken = { userId: data.id };
                 const cookie = sign(userToken, process.env.SECRET_KEY);
                 res
                   .cookie('applicant', cookie, { httpOnly: true })
-                  .json({ data });
+                  .json({ status: 'successfully', role: 'applicant', data });
```

Comment:
you send to front-end all user data from database, maybe it is not secure to return all user data,
![image](https://user-images.githubusercontent.com/32321796/78508962-5a790580-7793-11ea-8388-a7db42a2239c.png)

Reply:
I think that is the purpose of login.